example/loaders/load_forced.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
```

Comment:
I find forced to be ambiguous, so would strongly prefer another filename. E.g. "forced override of X value".

Reply:
Don't quite follow what you mean by 'forced override of X value'?